src/averageAges.js

```diff
@@ -34,7 +38,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people
```

Comment:
filter people array by right condition (use ternary operator) depend on withChildren parameter, don't repeat yourself.
if you can find the same piece of code in your work more than one-time that's mean it need to be refactored and repetition should be removed